.travis.yml

```diff
@@ -37,9 +37,8 @@ before_script: if [ "$VAADIN_CHARTS_LICENSE_CODE" != "" ]; then
                   echo "$VAADIN_CHARTS_LICENSE_CODE" > ~/.vaadin.charts.developer.license;
                fi;
 
-# as agreed in our SOP, build everything (don't deploy, just try to 'mvn package' locally, which covers unit tests)
-script: mvn build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.incrementalVersion}-SNAPSHOT versions:update-child-modules && mvn versions:commit &&
-        mvn --quiet --activate-profiles !development-build,!release-build --settings .travis.settings.xml clean cobertura:cobertura package
+# as agreed in our SOP, build everything (don't deploy, just try to 'mvn install' locally, which covers unit tests)
+script: mvn --quiet --activate-profiles !development-build,!release-build --settings .travis.settings.xml clean install
```

Comment:
What did the previous code do? Seems to me that it also checked the versioning?

Reply:
I removed it, as it was not part of the template. I added it earlier, because I think that a human should not take care of removing SNAPSHOT tags. This can be done by Maven. Also, using the goal `install` fixed the broken travis build